assets/scss/navbar-crafted.scss

```diff
@@ -1,10 +1,147 @@
 @use 'variables/colors' as *;
 @use 'utils' as *;
+@use 'variables/breakpoints' as *;
 
 header {
   background-color: $crafted-bg;
+  @include flex;
+  align-items: center;
+  .crafted__arrow {
+    color: white;
+    background-color: inherit;
+    border: 1px solid rgba($color: white, $alpha: 0.2);
+    border-radius: 5px;
+    padding: 5px;
+    font-size: 1.5rem;
+  }
+  .crafted__left-arrow {
+    margin-left: 40px;
+  }
+  .crafted__right-arrow {
+    margin-right: 40px;
+  }
 }
 
 .navbar {
   @include flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 0;
+  ul {
+    list-style-type: none;
+    @include flex;
+  }
+  .lni-menu {
+    font-size: 1.5rem;
+    // margin-left: auto;
+
+    color: white;
+  }
+  img {
+    width: 60px;
+  }
+  button {
+    // margin-left: auto;
+    padding: 12px;
+    border-radius: 5px;
+    border: none;
+    font-weight: 700;
+    a {
+      color: $crafted-bg;
+      text-transform: uppercase;
+      font-size: 0.8rem;
+    }
+  }
+  .navbar__links {
+    @include flex;
+    li a {
+      padding: 0 15px;
+      color: white;
+      text-transform: uppercase;
+      font-weight: 700;
+      font-size: 0.8rem;
+    }
+  }
+}
+
+.crafted {
+  color: white;
+  padding: 80px 0;
+  @include flex;
+  align-items: center;
+  @media (min-width: $extra-small) {
+    padding: 40px 0 10px 0;
+  }
+  h1:nth-of-type(1) {
+    font-size: 4rem;
+  }
+  h1:nth-of-type(2) {
+    font-size: 2rem;
+    padding: 0 80px 30px 0;
+  }
+  p {
+    padding: 0 30px 30px 0;
+    @media (min-width: $extra-small) {
+      padding: 20px 80px 20px 0;
+    }
+  }
+  img {
+    width: 500px;
+  }
+}
+
+.crafted__buttons {
+  padding: 20px 0;
+
+  button {
+    border-radius: 30px;
+    padding: 12px;
+    text-transform: uppercase;
+    border: 2px solid white;
+    font-weight: 700;
+  }
+  .crafted__get-started {
+    background-color: white;
+    color: $crafted-bg;
+    margin-right: 5px;
+    @media (min-width: $extra-small) {
+      margin-right: 15px;
+    }
+  }
+  .crafted__download {
+    background-color: inherit;
+    color: white;
+  }
+}
+
+// Small Screen
+
+@media only screen and (max-width: $extra-small) {
```

Comment:
I don't think you have to specify this media query.
You can put all of it outside this query since are designing mobile first, and then the media query for large screen defines the style for when not on mobile.

Reply:
moved to issue #39 
refractor later, will merge for now